### DISCUSSION TITLE
Where is the folder of the Images ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [requarks/wiki#6058] [Help / Questions] [answered]
Hello ,

I managed to move my database to a new server , I have already a backup of images and html pages.

The question is where I can find images folder ? so I can copy the files to this folder ? 

Best Regards

**Comment by fnafox** [comment, marked as answer]
Everything is stored in the database. Moving the database and starting the server is all that's needed to resume functionality.
If you explicitly want to manually dump content to disk, under Admin section select Storage > Local Filesystem and run the Dump all content to disk function.

**Reply by Jason-MK** [reply]
Thanks a lot @fnafox
